src/main/java/com/endava/soa_5ed/task2/arrays/MaxNumberIdentifier.java

```diff
@@ -0,0 +1,17 @@
+package com.endava.soa_5ed.task2.arrays;
+
+public class MaxNumberIdentifier {
+    /* Arrays - Exercise 3 */
+    public int getIndexOfMax(int[] array) {
+        return indexOfMax(array);
+    }
+
+    private int indexOfMax(int[] array) {
+        int maxIndex = 1;
```

Comment:
To avoid iteration repetition or comparing between the same indexes 
- (array[0] > array[1]) = true -> will result that these indexes will be verified one more time in next iteration (array[1] > array[0])
- (array[0] > array[1]) = false -> will result that in next iteration will be compared the same index 1 (array[1] > array[1])
better to start the iteration with 1st array index by giving zero as maxIndex value.

Reply:
good point, thank you, changed it